courses/fundamentals_of_ada/070_subprograms.rst

```diff
@@ -77,136 +76,118 @@ A Little "Preaching" About Names
    function Square_Root (V: Real) return Real;
    function Is_Open (V: Valve) return Boolean;
 
-========================
-Declarations and Bodies
-========================
-
-----------
-Examples
-----------
-
-.. include:: examples/070_subprograms/declarations_and_bodies.rst
+========
+Syntax
+========
 
-:url:`https://learn.adacore.com/training_examples/fundamentals_of_ada/070_subprograms.html#declarations-and-bodies`
+-------------------------
+ Specification and Body
+-------------------------
 
---------------------------
-Subprogram Declarations
---------------------------
+* Subprogram specification is the external (user) **interface**
 
-* Define the external (user) interface
+   - **Declaration** and **specification** are used synonymously
 
-   - Provide the subprogram specification
-   - *Declaration* and *specification* are often used synonymously
+* Specification may be required
 
-* Required in some circumstances
+   - Package specification
+   - Recursion
+   - Optional otherwise
```

Comment:
This last bullet isn't grammatically the same as the other two - maybe put parentheses around it?